# set Trajectory type
uint8 TRAJECTORY_STRAIGHT_LINE = 0
uint8 TRAJECTORY_CIRCLE_2D = 1
uint8 TRAJECTORY_CIRCLE_3D = 2
uint8 TRAJECTORY_ELLIPSE_2D = 3
uint8 TRAJECTORY_ELLIPSE_3D = 4

uint8 type
geometry_msgs/Vector3 center
geometry_msgs/Vector3 radius
geometry_msgs/Vector3 phase
---
bool success